__snapshots__/flex-list.test.js

```diff
@@ -1,20 +1,29 @@
-exports['flex list by specifying a profile and valid options (app and id) should succeed 1'] = `
-debug:  Checking for package updates
-debug:  Request:  GET http://localhost:3234/v2/apps/123/data-links
-debug:  Response: GET http://localhost:3234/v2/apps/123/data-links 200
-info:  You have 1 Kinvey service connectors:
-info:  TestKinveyDatalink
-info:  The service used in this project is marked with *
+exports['flex list by specifying a profile and valid options (app and id) should succeed  and output default format 1'] = `
+[debug] Checking for package updates
+[debug] Request:  GET http://localhost:3234/v2/apps/123/data-links
+[debug] Response: GET http://localhost:3234/v2/apps/123/data-links 200
+Count: 1
```

Comment:
It would unnecessarily complicate building the result and its output IMO. Moreover, if you run `profile list` (or any other `list` command) I think it's clear what is the count of.

Reply:
Not convinced about this, but OK... for now.